### PR TITLE
fix: Electron 3.0.6

### PR DESCRIPTION
**Pull request by pwnsdx**
- Fixed account switching shortcut
- Fixed environment swapping (via saving settings before leaving the app)
- Fixed WEBAPP-5587
- Refactored App.jsx

### CODE REVIEW

File: electron/src/js/menu/system.ts [hunk 1]
```diff
@@ -399,6 +399,20 @@ const createMenu = (isFullScreen: boolean): Menu => {
     helpTemplate.submenu = [];
   }
 
+  // Global mute shortcut
+  globalShortcut.register('CmdOrCtrl+Alt+M', () => sendAction(EVENT_TYPE.CONVERSATION.TOGGLE_MUTE));
+
+  // Global account switching shortcut
+  const switchAccountShortcut = ['CmdOrCtrl', 'Super'];
+  const accountLimit = 3;
```
Comment (bennyn) [inline electron/src/js/menu/system.ts]: Is the account limit specified somewhere else? I could imagine we have it as a config setting already?

Comment (lipis) [inline electron/src/js/menu/system.ts]: is that a hard limit?

File: package.json [hunk 2]
```diff
@@ -58,7 +58,7 @@
   },
   "license": "LicenseRef-LICENSE",
   "lint-staged": {
-    "*.{js,jsx}": [
+    "*.{js}": [
```
Comment (bennyn) [inline package.json]: I will fix the JSX styling issue.

Comment (lipis) [reply package.json]: @pwnsdx you can use the `--no-verify` if that is causing some errors..

File: electron/src/js/menu/system.ts (hunk 1 above)
Comment (bennyn) [reply electron/src/js/menu/system.ts]: yes, it is

**Pull request merged**